dvc/commands/imp_db.py

```diff
@@ -65,61 +72,87 @@ def run(self):
 
 class CmdImportDb(CmdBase):
     def run(self):
-        if not (self.args.sql or self.args.model):
-            raise argparse.ArgumentTypeError("Either of --sql or --model is required.")
+        from dvc.exceptions import InvalidArgumentError
+
+        if self.args.table or self.args.sql:
```

Comment:
Are one of these required?

Reply:
Yes they are required, and are mutually exclusive.